addons/gunbag/XEH_preInit.sqf

```diff
@@ -24,4 +24,22 @@ PREP_RECOMPILE_END;
     }, _this] call CBA_fnc_execNextFrame;
 }] call CBA_fnc_addClassEventHandler;
 
+[QEGVAR(arsenal,displayOpened), {
+
+    private _center = EGVAR(arsenal,center);
+
+    if (_center call FUNC(hasGunBag)) then {
+        GVAR(arsenalCache) = (backpackContainer _center) getVariable [QGVAR(gunbagWeapon), []];
```

Comment:
Maybe use a more descriptive varname. As this always stores the gunbag. Why not gunbagCache?
You might want to use arsenalCache for something like an actual arsenalCache later on.

Reply:
the full var is:
ace_gunbag_arsenalCache, I don't really know if there's a more descriptive varname :/

Reply:
ooooohhhh. duh. Should've looked at what file that is. Expected it to be in arsenal